internal/tensorzero-node/lib/bindings/UpdateChatDatapointRequest.ts

```diff
@@ -0,0 +1,41 @@
+// This file was generated by [ts-rs](https://github.com/Aleph-Alpha/ts-rs). Do not edit this file manually.
+import type { ContentBlockChatOutput } from "./ContentBlockChatOutput";
+import type { DatapointMetadataUpdate } from "./DatapointMetadataUpdate";
+import type { Input } from "./Input";
+import type { ToolCallConfigDatabaseInsert } from "./ToolCallConfigDatabaseInsert";
+
+/**
+ * An update request for a chat datapoint.
+ * For any fields that are optional in ChatInferenceDatapoint, the request field distinguishes between an omitted field, `null`, and a value:
+ * - If the field is omitted, it will be left unchanged.
+ * - If the field is specified as `null`, it will be set to `null`.
+ * - If the field has a value, it will be set to the provided value.
+ *
+ * In Rust this is modeled as an `Option<Option<T>>`, where `None` means "unchanged" and `Some(None)` means "set to `null`" and `Some(Some(T))` means "set to the provided value".
+ */
+export type UpdateChatDatapointRequest = {
+  /**
+   * The ID of the datapoint to update. Required.
+   */
+  id: string;
+  /**
+   * Datapoint input. If omitted or `null`, it will be left unchanged.
+   */
+  input?: Input;
+  /**
+   * Chat datapoint output. If omitted, it will be left unchanged. If `null`, it will be set to `null`. If specified as a value, it will be set to the provided value.
+   */
+  output?: Array<ContentBlockChatOutput> | null;
+  /**
+   * Datapoint tool parameters. If omitted, it will be left unchanged. If specified as `null`, it will be set to `null`. If specified as a value, it will be set to the provided value.
+   */
+  tool_params?: ToolCallConfigDatabaseInsert | null;
+  /**
+   * Datapoint tags. If omitted, it will be left unchanged. If specified as `null`, it will be set to `null`. If specified as a value, it will be set to the provided value.
+   */
+  tags?: { [key in string]?: string } | null;
```

Comment:
this is wrong; if it's null we should not touch it. The empty map should clear and null should leave it alone.

Reply:
Yep, fixed.